lint-staged.config.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+    '*.{json,md,yml,css}': ['prettier --write'],
+    '*.{js,ts}': ['eslint --fix'],
```

Comment:
Can't we use prettier to format js and ts too?

Reply:
We use prettier as eslint plugin

Reply:
Makes sense, thanks